src_kolko_krzyżyk/kolko.py

```diff
@@ -0,0 +1,76 @@
+class TicTacToe:
+    #class veribles
```

Comment:
I guess you wanted to say `variables` not `veribles`. This comment says nothing anyway.